users/models.py

```diff
@@ -1,18 +1,39 @@
+"""@package docstring
+User model.
+
+@author     7Pros
+@copyright  Some license
+"""
```

Comment:
This should be in a separate PR, as it is not related to the hashtag feature in any way.